.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly make sure to add `node_modules` also as per the requirement of this project.

Reply:
fixed in https://github.com/teckim/hello-microverse/pull/1/commits/308960c3186b5860a12912b604626e96ca6114e4